gammapy/modeling/models/tests/test_spectral.py

```diff
@@ -437,26 +437,34 @@ def test_model_plot_sed_type():
     with mpl_plot_check():
         ax1 = pwl.plot((1 * u.TeV, 100 * u.TeV), sed_type="dnde")
         ax2 = pwl.plot_error((1 * u.TeV, 100 * u.TeV), sed_type="dnde")
-        assert ax1.axes.axes.get_ylabel() == "dnde [1 / (cm2 s TeV)]"
-        assert ax2.axes.axes.get_ylabel() == "dnde [1 / (cm2 s TeV)]"
+        assert ax1.yaxis.units.is_equivalent(u.Unit("1 / (s cm2 TeV)"))
+        assert ax1.axes.axes.get_ylabel().split()[0] == "dnde"
+        assert ax2.yaxis.units.is_equivalent(u.Unit("1 / (s cm2 TeV)"))
+        assert ax2.axes.axes.get_ylabel().split()[0] == "dnde"
 
     with mpl_plot_check():
         ax1 = pwl.plot((1 * u.TeV, 100 * u.TeV), sed_type="e2dnde")
         ax2 = pwl.plot_error((1 * u.TeV, 100 * u.TeV), sed_type="e2dnde")
-        assert ax1.axes.axes.get_ylabel() == "e2dnde [erg / (cm2 s)]"
-        assert ax2.axes.axes.get_ylabel() == "e2dnde [erg / (cm2 s)]"
+        assert ax1.yaxis.units.is_equivalent(u.Unit("erg / (cm2 s)"))
```

Comment:
I remember I had some discussions with Christoph in the past on how to test something like this. In the end he convinced me that one should test, that the unit does not change at all. E.g. in our code (for what ever reason...) the units could change to `"TeV / (m2 s)` and the test would pass.

So my proposal here would be to just cover both cases:
```python3
label = ax2.axes.axes.get_ylabel()
assert label in ["e2dnde [erg / (cm2 s)]", "..." ]
```

Reply:
> Do you know if we can check if units are same, ie, not equivalent?

Just comparing the units seems to work as expected:

```python
from astropy import units as u

a = u.Unit("erg / (cm2 s)")
b = u.Unit("erg / (s cm2)")
c = u.Unit("TeV / (s cm2)")

print(a == b)
print(a == c)
```